### PR TITLE
Sls App filter changes

### PR DESCRIPTION
**Pull request by swati-sh**
### Requirements

 Filter changes for all service page
### Description of the Change
Added Asset Type Filter and sls app

### CODE REVIEW

File: core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts [hunk 1]
```diff
@@ -29,21 +51,18 @@ export class FilterModalComponent implements OnInit {
...
-  }
-
-  reset() {
-    this.ngOnInit();
-  }
-
+    this.form.columns = columns;
+  }}}
   changeFilter(filterSelected, filterField) {
+    console.log("filterSelected, filterField",filterSelected, filterField)
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts]: remove console.log

File: core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts [hunk 2]
```diff
@@ -1,25 +1,47 @@
...
 
   constructor() {
   }
-
   ngOnInit() {
+    // console.log("fields",this.fields)
+    // let columns = _(this.fields)
+    //   .groupBy('column')
+    //   .map((column, key, array) => {
+    //     return {
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts]: remove commented lines

File: core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts [hunk 3]
```diff
@@ -1,25 +1,47 @@
-import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {Component, EventEmitter, Input, OnInit, Output,OnChanges, SimpleChanges, SimpleChange} from '@angular/core';
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts]: remove onchanges, simplechanges if not used

File: core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts [hunk 4]
```diff
@@ -384,17 +383,20 @@ export class CreateServiceComponent implements OnInit {
     }
 
     this.isLoading = true;
+    console.log(payload)
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts]: remove all consoles from this files

File: core/jazz_ui/src/app/secondary-components/advanced-filters/OSS/advanced-filters.component.ts [hunk 5]
```diff
@@ -15,13 +15,10 @@ export class AdvancedFiltersComponentOSS implements OnInit {
...
     @Input() logs:boolean = false;
     @Input() assets:boolean = false;
-
     @Input() service: any = {};
-
-    
     @Output() onFilterSelect:EventEmitter<any> = new EventEmitter<any>();
-
-
+    //@Input() assetSelected;
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/advanced-filters/OSS/advanced-filters.component.ts]: remove commented lines

File: core/jazz_ui/src/app/primary-components/radio-group/radio-group.component.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
+ @import './../../../main-styles';
+@import'../../secondary-components/create-service/create-service.component.scss';
+//   input[type='radio']:checked:after {
+//     width: 15px;
+//     height: 15px;
+//     border-radius: 15px;
```
Comment (SauarvD) [inline core/jazz_ui/src/app/primary-components/radio-group/radio-group.component.scss]: remove commented lines

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts [hunk 7]
```diff
@@ -249,6 +279,7 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
   }
 
   queryMetricsData() {
+    debugger
```
Comment (SauarvD) [inline core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: remove debugger

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts [hunk 8]
```diff
@@ -207,11 +216,21 @@ export class ServiceLogsComponent implements OnInit {
...
+		var asset = asset_type_list.env[index];
+		this.asset_type = event;
+		this.payload.asset_type=asset;
+		this.resetPayload();
+	 }
  
 	// onEnvSelected(env){
 
 	onEnvSelected(envt){
+		debugger
```
Comment (SauarvD) [inline core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: remove debugger and console from this file

File: core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts [hunk 9]
```diff
@@ -498,13 +501,17 @@ export class EnvLogsSectionComponent implements OnInit {
...
+			this.instance_yes.onAssetListSelected('Dynamo Db');
+
+		  }
 		  case 'all':{ this.instance_yes.onRangeListSelected('Day');    
 				this.instance_yes.onPeriodSelected('15 Minutes');
 				this.instance_yes.onStatisticSelected('Average');
 				this.instance_yes.onaccSelected('Acc 1');
 				this.instance_yes.onregSelected('reg 1');
 				this.instance_yes.onEnvSelected('prod');
-				this.instance_yes.onMethodListSelected('POST');
```
Comment (SauarvD) [inline core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts]: make sure if this is required or not

File: core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts (hunk 1 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts]: removed

File: core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts (hunk 2 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts]: removed

File: core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts (hunk 3 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts]: removed

File: core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts (hunk 4 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts]: removed

File: core/jazz_ui/src/app/secondary-components/advanced-filters/OSS/advanced-filters.component.ts (hunk 5 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/advanced-filters/OSS/advanced-filters.component.ts]: removed

File: core/jazz_ui/src/app/primary-components/radio-group/radio-group.component.scss (hunk 6 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/primary-components/radio-group/radio-group.component.scss]: removed

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts (hunk 7 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: removed

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts (hunk 8 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: removed

File: core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts (hunk 9 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.ts]: checked

File: core/jazz_ui/src/environments/environment.oss.ts [hunk 10]
```diff
@@ -1,15 +1,71 @@
...
+        "deployments": true,
+        "code_quality": "false",
+        "logs": true,
+        "assets": true
+    }
+},
   configFile: 'config/config.oss.json',
-  baseurl: "https://{API_GATEWAY_KEY_PROD}.execute-api.{inst_region}.amazonaws.com/prod",
-  api_doc_name: "https://{api_doc_name}.s3.amazonaws.com",
+  baseurl: "https://lp5zztke62.execute-api.us-east-1.amazonaws.com/prod",
```
Comment (supritat) [inline core/jazz_ui/src/environments/environment.oss.ts]: revert this to variables.

Comment (swati-sh) [reply core/jazz_ui/src/environments/environment.oss.ts]: Done

File: core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts [hunk 11]
```diff
@@ -384,17 +383,20 @@ export class CreateServiceComponent implements OnInit {
...
     this.http.post('/jazz/create-serverless-service' , payload)
         .subscribe(
         (Response) => {
           var output = Response;
+          console.log("res",output)
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts]: remove consoles from this file

File: core/jazz_ui/src/app/secondary-components/filter-tags/filter-tags.component.ts [hunk 12]
```diff
@@ -64,13 +72,15 @@ export class FilterTagsComponent implements OnInit {
...
         filter_Method_default:any='POST';
+        filter_Asset_default:any = 'lambda';
 
 
     constructor(private cache: DataCacheService){
 
     }
 
     setDefaults(){
+        console.log("this.selected",this.selected)
```
Comment (SauarvD) [inline core/jazz_ui/src/app/secondary-components/filter-tags/filter-tags.component.ts]: remove console

Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/filter-tags/filter-tags.component.ts]: removed

File: core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts (hunk 11 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts]: removed

**Comment by supritat** [comment]
#964 includes these changes.

**Pull request closed without merging**